spec-main/api-desktop-capturer-spec.ts

```diff
@@ -46,10 +46,8 @@ ifdescribe(features.isDesktopCapturerEnabled() && !process.arch.includes('arm')
     expect(promise2).to.not.eventually.be.rejected()
   })
 
-  it('returns an empty display_id for window sources on Windows and Mac', async () => {
-    // Linux doesn't return any window sources.
-    if (process.platform !== 'win32' && process.platform !== 'darwin') return
-
+  // Linux doesn't return any window sources.
+  ifit(process.platform !== 'linux')('returns an empty display_id for window sources on Windows and Mac', async () => {
```

Comment:
https://github.com/electron/electron/blob/b8e66830d7746bad1d1ff7b56b6d029294849605/spec-main/api-desktop-capturer-spec.ts#L9 - The whole test is skipped for Windows so we really only need to check for Darwin here and/or enable Windows for the whole test.

Reply:
I'm trying to stick to the old logic, presuming that there's some sort of reason things were the way they were. Ideally I'd like to re-enable this test suite on Windows, but I think that should be a followup PR.

Reply:
👍